RubberduckCodeAnalysis/ChainedWrapperAnalyzer.cs

```diff
@@ -39,9 +39,7 @@ private static void AnalyzeSymbol(SyntaxNodeAnalysisContext context)
             }
 
             var expInterfaces = context.SemanticModel.GetTypeInfo(node.Expression).Type?.AllInterfaces;
-
-            var nameValue = node.Name.Parent.Parent is InvocationExpressionSyntax ? node.Name.Parent.Parent : node.Name;
-            var nameInterfaces = context.SemanticModel.GetTypeInfo(nameValue).Type?.AllInterfaces;
+            var nameInterfaces = context.SemanticModel.GetTypeInfo(node.Name).Type?.AllInterfaces;
```

Comment:
What's up with the changed logic here?

Reply:
The `nameValue` is never used...

Reply:
Ummm... `var nameInterfaces = context.SemanticModel.GetTypeInfo(nameValue).Type?.AllInterfaces;`